shogun-keycloak/themes/shogun/login/resources/css/styles.css

```diff
@@ -0,0 +1,101 @@
+.login-pf body {
+
+  background: white;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```

Comment:
Is this font also used in the other forms, e.g. the password reset one?

Reply:
yes